src/io/linux.zig

```diff
@@ -188,6 +199,18 @@ pub const IO = struct {
                 },
                 else => return err,
             };
+
+            self.ios_in_kernel += self.ios_queued;
```

Comment:
Should this use what's returned by `submit_and_wait()` above? Could be less than `ios_queued`.

Reply:
Good point.

If I do:

```
const submitted = ...;
self.ios_queued -= submitted;
self.ios_in_kernel += submitted;
```

Then the `-=` underflows because we're also counting timer submits. Separating them seems hard so I just started counting timers too.